datafusion-cli/src/main.rs

```diff
@@ -44,7 +44,7 @@ pub async fn main() {
         )
         .arg(
             Arg::with_name("batch-size")
-                .help("The batch size of each query, default value is 1048576")
```

Comment:
We could get this value programmatically by creating a default config and getting this value.

Reply:
:+1: agreed

Reply:
I made it inherit the DataFusion default now.